CONTRIBUTING.md

```diff
@@ -21,18 +21,166 @@ To get an overview of the project, read the [README](README.md). Here are some r
+
+##### **Adding Other Content**
+ - **Audio Files**:
+ You can embed audio files using the HTML <audio> tag.
+ - **Interactive Content with HTML Widgets**:
+ Quarto supports various HTML widgets that can add interactive content to your document. 
+
+##### Tips 
+- **Relative Paths**:
+Use relative paths for local files to ensure your document remains portable. 
+
+- **Test Across Browsers**: Ensure multimedia content works across different browsers and devices. 
+
+- **Output Formats**: Verify that multimedia elements render correctly in the intended output formats (HTML, PDF, etc.). 
+
+By following these methods, you can effectively add videos, images, audio, and interactive content to your Quarto documents, enhancing their informativeness and engagement.
+
 
 #### Redirecting pages where changing / removing
 
+Redirecting pages in a Quarto project for R involves creating ``a _quart.yml`` configuration file where you can define redirects.
+
+1. **``Create/Edit_quarto.yml``** :
+Ensure you have a ``_quarto.yml`` file in your Quarto project.
+
+2. **``Add redirects``**:
+In ``_quarto.yml``, define redirects like this:
+                                    - From: ``old-page.html``
+                                    to:     ``new-page.html``
+                                  
+3. **``Render Project``** :
+Run ``quarto render`` in your project directory to apply changes.
+
+4.**``Verify``**:
+Test old URLs to ensure they redirect correctly.
+
+This sets up URL redirects for pages that have changed or been removed in your Quarto project.
+
 #### Checking for broken links when changing headings
 
+When changing headings in Quarto project, it's crucial to ensure that all links referencing these headings are still correct. Here's a quick guide to check for broken links:
+
+1. **Change Headings**: Update your headings in the relevant ``.qmd files``.
+
+2. **Run Quarto's Built-in Link Checker**: Quarto has a built in feauture to check for broken links. 
+
+Run the following command in your project directory: 
+
+```quarto check ```
+
+This command will scan your project for broken links and report any issues. 
+
+3. **Manually Update Links**: If any broken links are reported, update them to match the new headings. Use the correct anchors, which typically follow the format `#new-heading-text`. 
+
+4. **Re-render Project**: After updating links, re-render your project: ```quarto render ``` 
+
+5. **Double-Check**: Manually verify a few pages to ensure that the links work as expected. 
+
+### Example 
+
+If you changed a heading from `## Old Heading` to `## New Heading`, make sure all links like 
+`[link text](#old-heading)` are updated to `[link text](#new-heading)`. 
+By following these steps, you can efficiently check for and fix broken links when changing headings in your Quarto project.
+
 #### Building locally before raising a pull request
 
+Here's a streamlined guide for building your Quarto project locally before raising a pull request: 
+
```

Comment:
It's probably worth highlighting that these commands are run from the terminal and not the console.